### PR TITLE
#1201 - Disable drawing charts in multisim to speed it up

### LINKED ISSUE #1201: [Simplots] Disable drawing charts in multisim to speed it up
Disable drawing charts in multisim to speed it up

### PR DESCRIPTION
**Pull request by trizin**
Fixes #1201

Changes proposed in this PR:

- Allow disabling state updates by calling a function in sim engine.
- Disable state updates in multisim.

### CODE REVIEW

File: pdr_backend/sim/multisim_engine.py [hunk 1]
```diff
@@ -69,6 +69,7 @@ async def run_one(self, run_i: int):
         feedset = ppss.predictoor_ss.predict_train_feedsets[0]
         multi_id = str(uuid.uuid4())
         sim_engine = SimEngine(ppss, feedset, multi_id)
+        sim_engine.disable_realtime_state()
```
Comment (trentmc) [inline pdr_backend/sim/multisim_engine.py:72]: Don't you need to re-enable it at the end?

File: pdr_backend/sim/sim_engine.py [hunk 2]
```diff
@@ -44,6 +44,9 @@ def __init__(
 
         self.ppss = ppss
 
+        # can be disabled by calling disable_realtime_state()
+        self.state_updates = True
```
Comment (trentmc) [inline pdr_backend/sim/sim_engine.py:48]: Rename `state_updates` -> `do_state_updates`. More obvious.

**Review by trentmc — CHANGES_REQUESTED** [review]

File: pdr_backend/sim/sim_engine.py [hunk 3]
```diff
@@ -328,12 +331,19 @@ def _sell(self, price: float, tokcoin_amt_send: float) -> float:
 
         return usdcoin_amt_recd
 
+    def disable_realtime_state(self):
+        self.state_updates = False
+
     @enforce_types
     def save_state(self, i: int, N: int):
         "Save state on this iteration Y/N?"
```
Comment (calina-c) [inline pdr_backend/sim/sim_engine.py:339]: is this comment a TODO? If so, please mark it as such. I know that pylint breaks, but that's the point. We should not merge anything that has questions pending.

Comment (trentmc) [reply pdr_backend/sim/sim_engine.py:339]: @calina-c Not sure what you refer to, I don't see any comment.

File: pdr_backend/sim/multisim_engine.py (hunk 1 above)
Comment (trentmc) [reply pdr_backend/sim/multisim_engine.py:72]: Ah, right! Thanks.

**Review by trentmc — APPROVED** [review]

**Pull request merged**